test/convert.bats

```diff
@@ -64,3 +64,17 @@ EOF
   rm -f stacker.yaml stacker-subs.yaml
   stacker clean
 }
+
+@test "build python image" {
+  git clone https://github.com/docker-library/python.git
+  chmod -R a+rwx python
```

Comment:
this seems questionable.
you really want to make *everything* under there executable?

Reply:
fixed

Reply:
mkdir /home/runner/work/stacker/stacker/stackertest-test_build_python_image.2Okow7/python/3.11/alpine3.17/roots: permission denied couldn't make rootfs dir for testsuite

^ but this. Fixing.

Reply:
is that an indication of a larger problem with permissions? It seems unlikely that the problem here is with that specific directory and not more generic.

stacker should only need write permission for the uid owner of that directory in order to create 'roots' (and .stacker and oci) in the directory. 

what was the ownership of alpine3.17/ ?  How does content for the test get copied into that temp dir? maybe umask got foobar'ed somehow?

Reply:
Let's find out.

Reply:
```
# Cloning into 'python'...
# python/:
# total 104
# drwxr-xr-x 10 root   root   4096 Feb 14 17:37 .
# drwx------  5 runner runner 4096 Feb 14 17:37 ..
# drwxr-xr-x  8 root   root   4096 Feb 14 17:37 .git
# -rw-r--r--  1 root   root     94 Feb 14 17:37 .gitattributes
# drwxr-xr-x  3 root   root   4096 Feb 14 17:37 .github
# -rw-r--r--  1 root   root     17 Feb 14 17:37 .gitignore
```

Everything is root-owned, that's why